### PR TITLE
Remove stale note in loki.source.podlogs

### PR DESCRIPTION
**Pull request by clayton-cornell**
#### PR Description
Remove stale note in loki.source.podlogs

#### PR Checklist

- [ ] CHANGELOG.md updated
- [ ] Documentation added
- [ ] Tests updated
- [ ] Config converters updated

### CODE REVIEW

File: docs/sources/reference/components/loki.source.podlogs.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ title: loki.source.podlogs
 
 `loki.source.podlogs` discovers `PodLogs` resources on Kubernetes and, using
 the Kubernetes API, tails logs from Kubernetes containers of Pods specified by
-the discovered them.
+the discovered Pods.
```
Comment (clayton-cornell) [inline docs/sources/reference/components/loki.source.podlogs.md:11]: Is this correct? The sentence was incomplete....

Comment (rfratto) [reply docs/sources/reference/components/loki.source.podlogs.md:11]: I think I meant to say:

> `loki.source.podlogs` discovers `PodLogs` resources on Kubernetes and, using the Kubernetes API, tails logs from Kubernetes containers of Pods specified by the PodLogs resource. 

Basically: 

* `loki.source.podlogs` discovers PodLogs resources 
* PodLogs resources give rules for which Pods to discover on your cluster 

`loki.source.podlogs` _indirectly_ discovers Pods, but only based on the rules in the directly discovered PodLogs resources.

**Review by rfratto — APPROVED** [review]
LGTM for removing the admonition, but I think the intro paragraph needs a bit more tweaking still

File: docs/sources/reference/components/loki.source.podlogs.md (hunk 1 above)
Comment (clayton-cornell) [reply docs/sources/reference/components/loki.source.podlogs.md]: Updated the intro text as we discussed.

**Pull request merged**